src/_includes/dart-tool.md

```diff
@@ -0,0 +1,38 @@
+{{site.alert.note}}
+  As of Flutter's 1.19.0 dev release, the `dart` command
+  is now surfaced for you. Downloading the Flutter SDK
+  also downloads the compatible version of Dart,
+  but if you have downloaded the Dart SDK separately,
+  make sure that the Flutter version of `dart` is listed
+  first in your path, as the two versions may not be compatible.
+  The following command (on macOS, Linux, and Chrome OS),
+  tells you whether the `flutter` and `dart` commands
+  originate from the same `bin` directory and are therefore
+  compatible. (Some versions of Windows
+  support a similar `where` command.)
+
+  ```sh
+  $ which flutter; which dart
+  <path-to-flutter-sdk>/flutter-sdk/bin/flutter
+  /usr/local/bin/dart
+  ```
+
+  As shown above, the two commands are not coming from
+  the same `bin` directory. Update your path to use
+  commands from `<path-to-flutter-sdk>/bin` before it
+  uses commands from `/usr/local/bin` (in this case).
+  After updating your shell for the change to take effect,
+  run the command again, and you should now see:
+
+  ```sh
+  $ which flutter; which dart
```

Comment:
It'd be nice to have a non-Unix version of this text for our Windows installation page, so our Windows users (who might be the majority?) can see the right command and path. But it shouldn't derail us. Shall I file a bug?

Reply:
I searched and searched for a Windows equivalent. If you can find a direct, not-too-complex equivalent, I'll add it. On Windows, this is a much more complicated operation.

Reply:
I figure we can let the community help us find the answer here. I could file a bug and mark it **help wanted**.

Reply:
I'd love that! I was tempted to put that request right into the text, but asking via the bug is better. Also put PRs welcome.  :D

Reply:
#4193